src/bootstrap-table.js

```diff
@@ -2306,19 +2306,24 @@
     };
 
     BootstrapTable.prototype.updateByUniqueId = function (params) {
-        var rowId;
+        var me = this;
```

Comment:
Use `var that = this` in order to follow the standard. **Don't forget change the variable name in the function**